### PR TITLE
Added `black`'s check support.

### PR DESCRIPTION
**Pull request by jouyuy**
### Problem
Add support for the `black`'s check command (`black [source] --check`).

### Solution
Added `BlackCheckParser` for parsing `black`'s check output.

#### Automatic
```shell
$ pytest tests/test_parsers.py::BlackCheckParserTestCase
```

#### Manual
Not required.

#### Proof
Not required.

#### Dependent PRs
None.

### CODE REVIEW

**Comment by jouyuy** [comment]
Hey @grantmcconnaughey!

Could you please review this PR?

Thanks!

File: lintly/parsers.py [hunk 1]
```diff
@@ -183,6 +183,22 @@ def parse_violations(self, output):
         return violations
 
 
+class BlackCheckParser(BaseLintParser):
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Let's just call this `BlackParser`.

File: lintly/parsers.py [hunk 2]
```diff
@@ -211,4 +227,7 @@ def parse_violations(self, output):
     # lintly/static/sass/file1.scss
     #   13:1  ✖  Expected no more than 1 empty line   max-empty-lines
     'stylelint': StylelintParser(),
+
+    # Black's check command default formatter.
+    'black-check': BlackCheckParser(),
```
Comment (grantmcconnaughey) [inline lintly/parsers.py]: Let's make the option `black` instead of `black-check`.

**Comment by jouyuy** [comment]
Any time!

Yeah, your naming suggestion makes complete sense to me. I've addressed requested changes and also fixed broken test.

Thanks!

**Pull request merged**